templates/phaser/.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+indexer.db
+node_modules
```

Comment:
small nit: can we put `node_modules` first?

I tend to order my gitignore files by least-specific to most-specific

Reply:
Oh sure, will keep in mid in the future